spec/cypress/support/commands.js

```diff
@@ -22,6 +22,17 @@ Cypress.Commands.add("login", (...traits) => {
   cy.get('[action="/users/sign_in_with_token"] [name="commit"]').click();
 });
 
+Cypress.Commands.add("loginWithEmail", (email) => {
+  cy.visit("/users/sign_in");
+  cy.get("input[name*=email]").type(email);
+  cy.clickCommitButton();
+
+  cy.appEval(`User.find_by(email: "${email}")`).then((user) => {
```

Comment:
This won't work on accounts that don't have login tokens (such as the one in the test case on my PR)